bindings.c

```diff
@@ -4477,6 +4479,19 @@ static bool permute_root(void)
 	return true;
 }
 
+static int preserve_ns(int pid)
```

Comment:
This could be confusing later, could you name it preserve_mnt_ns() for precision?